public/api/conf/1.0/docs/scopes.md

```diff
@@ -2,7 +2,7 @@ Access to user information is controlled through scopes. Each access token (OAut
 
```

Comment:
Scopes.md can be deleted. All its content is included in the OAS YAML `info.description` property.

Reply:
was planning to delete all the legacy docs/files in a separate PR once its been validated